<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <title>Test Report</title>
    <style>body {
  font-family: Helvetica, Arial, sans-serif;
  font-size: 12px;
  /* do not increase min-width as some may use split screens */
  min-width: 800px;
  color: #999;
}

h1 {
  font-size: 24px;
  color: black;
}

h2 {
  font-size: 16px;
  color: black;
}

p {
  color: black;
}

a {
  color: #999;
}

table {
  border-collapse: collapse;
}

/******************************
 * SUMMARY INFORMATION
 ******************************/
#environment td {
  padding: 5px;
  border: 1px solid #E6E6E6;
}
#environment tr:nth-child(odd) {
  background-color: #f6f6f6;
}

/******************************
 * TEST RESULT COLORS
 ******************************/
span.passed,
.passed .col-result {
  color: green;
}

span.skipped,
span.xfailed,
span.rerun,
.skipped .col-result,
.xfailed .col-result,
.rerun .col-result {
  color: orange;
}

span.error,
span.failed,
span.xpassed,
.error .col-result,
.failed .col-result,
.xpassed .col-result {
  color: red;
}

/******************************
 * RESULTS TABLE
 *
 * 1. Table Layout
 * 2. Extra
 * 3. Sorting items
 *
 ******************************/
/*------------------
 * 1. Table Layout
 *------------------*/
#results-table {
  border: 1px solid #e6e6e6;
  color: #999;
  font-size: 12px;
  width: 100%;
}
#results-table th,
#results-table td {
  padding: 5px;
  border: 1px solid #E6E6E6;
  text-align: left;
}
#results-table th {
  font-weight: bold;
}

/*------------------
 * 2. Extra
 *------------------*/
.log {
  background-color: #e6e6e6;
  border: 1px solid #e6e6e6;
  color: black;
  display: block;
  font-family: "Courier New", Courier, monospace;
  height: 230px;
  overflow-y: scroll;
  padding: 5px;
  white-space: pre-wrap;
}
.log:only-child {
  height: inherit;
}

div.image {
  border: 1px solid #e6e6e6;
  float: right;
  height: 240px;
  margin-left: 5px;
  overflow: hidden;
  width: 320px;
}
div.image img {
  width: 320px;
}

div.video {
  border: 1px solid #e6e6e6;
  float: right;
  height: 240px;
  margin-left: 5px;
  overflow: hidden;
  width: 320px;
}
div.video video {
  overflow: hidden;
  width: 320px;
  height: 240px;
}

.collapsed {
  display: none;
}

.expander::after {
  content: " (show details)";
  color: #BBB;
  font-style: italic;
  cursor: pointer;
}

.collapser::after {
  content: " (hide details)";
  color: #BBB;
  font-style: italic;
  cursor: pointer;
}

/*------------------
 * 3. Sorting items
 *------------------*/
.sortable {
  cursor: pointer;
}

.sort-icon {
  font-size: 0px;
  float: left;
  margin-right: 5px;
  margin-top: 5px;
  /*triangle*/
  width: 0;
  height: 0;
  border-left: 8px solid transparent;
  border-right: 8px solid transparent;
}
.inactive .sort-icon {
  /*finish triangle*/
  border-top: 8px solid #E6E6E6;
}
.asc.active .sort-icon {
  /*finish triangle*/
  border-bottom: 8px solid #999;
}
.desc.active .sort-icon {
  /*finish triangle*/
  border-top: 8px solid #999;
}
</style></head>
  <body onLoad="init()">
    <script>/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */


function toArray(iter) {
    if (iter === null) {
        return null;
    }
    return Array.prototype.slice.call(iter);
}

function find(selector, elem) { // eslint-disable-line no-redeclare
    if (!elem) {
        elem = document;
    }
    return elem.querySelector(selector);
}

function findAll(selector, elem) {
    if (!elem) {
        elem = document;
    }
    return toArray(elem.querySelectorAll(selector));
}

function sortColumn(elem) {
    toggleSortStates(elem);
    const colIndex = toArray(elem.parentNode.childNodes).indexOf(elem);
    let key;
    if (elem.classList.contains('result')) {
        key = keyResult;
    } else if (elem.classList.contains('links')) {
        key = keyLink;
    } else {
        key = keyAlpha;
    }
    sortTable(elem, key(colIndex));
}

function showAllExtras() { // eslint-disable-line no-unused-vars
    findAll('.col-result').forEach(showExtras);
}

function hideAllExtras() { // eslint-disable-line no-unused-vars
    findAll('.col-result').forEach(hideExtras);
}

function showExtras(colresultElem) {
    const extras = colresultElem.parentNode.nextElementSibling;
    const expandcollapse = colresultElem.firstElementChild;
    extras.classList.remove('collapsed');
    expandcollapse.classList.remove('expander');
    expandcollapse.classList.add('collapser');
}

function hideExtras(colresultElem) {
    const extras = colresultElem.parentNode.nextElementSibling;
    const expandcollapse = colresultElem.firstElementChild;
    extras.classList.add('collapsed');
    expandcollapse.classList.remove('collapser');
    expandcollapse.classList.add('expander');
}

function showFilters() {
    const filterItems = document.getElementsByClassName('filter');
    for (let i = 0; i < filterItems.length; i++)
        filterItems[i].hidden = false;
}

function addCollapse() {
    // Add links for show/hide all
    const resulttable = find('table#results-table');
    const showhideall = document.createElement('p');
    showhideall.innerHTML = '<a href="javascript:showAllExtras()">Show all details</a> / ' +
                            '<a href="javascript:hideAllExtras()">Hide all details</a>';
    resulttable.parentElement.insertBefore(showhideall, resulttable);

    // Add show/hide link to each result
    findAll('.col-result').forEach(function(elem) {
        const collapsed = getQueryParameter('collapsed') || 'Passed';
        const extras = elem.parentNode.nextElementSibling;
        const expandcollapse = document.createElement('span');
        if (extras.classList.contains('collapsed')) {
            expandcollapse.classList.add('expander');
        } else if (collapsed.includes(elem.innerHTML)) {
            extras.classList.add('collapsed');
            expandcollapse.classList.add('expander');
        } else {
            expandcollapse.classList.add('collapser');
        }
        elem.appendChild(expandcollapse);

        elem.addEventListener('click', function(event) {
            if (event.currentTarget.parentNode.nextElementSibling.classList.contains('collapsed')) {
                showExtras(event.currentTarget);
            } else {
                hideExtras(event.currentTarget);
            }
        });
    });
}

function getQueryParameter(name) {
    const match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
    return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
}

function init () { // eslint-disable-line no-unused-vars
    resetSortHeaders();

    addCollapse();

    showFilters();

    sortColumn(find('.initial-sort'));

    findAll('.sortable').forEach(function(elem) {
        elem.addEventListener('click',
            function() {
                sortColumn(elem);
            }, false);
    });
}

function sortTable(clicked, keyFunc) {
    const rows = findAll('.results-table-row');
    const reversed = !clicked.classList.contains('asc');
    const sortedRows = sort(rows, keyFunc, reversed);
    /* Whole table is removed here because browsers acts much slower
     * when appending existing elements.
     */
    const thead = document.getElementById('results-table-head');
    document.getElementById('results-table').remove();
    const parent = document.createElement('table');
    parent.id = 'results-table';
    parent.appendChild(thead);
    sortedRows.forEach(function(elem) {
        parent.appendChild(elem);
    });
    document.getElementsByTagName('BODY')[0].appendChild(parent);
}

function sort(items, keyFunc, reversed) {
    const sortArray = items.map(function(item, i) {
        return [keyFunc(item), i];
    });

    sortArray.sort(function(a, b) {
        const keyA = a[0];
        const keyB = b[0];

        if (keyA == keyB) return 0;

        if (reversed) {
            return keyA < keyB ? 1 : -1;
        } else {
            return keyA > keyB ? 1 : -1;
        }
    });

    return sortArray.map(function(item) {
        const index = item[1];
        return items[index];
    });
}

function keyAlpha(colIndex) {
    return function(elem) {
        return elem.childNodes[1].childNodes[colIndex].firstChild.data.toLowerCase();
    };
}

function keyLink(colIndex) {
    return function(elem) {
        const dataCell = elem.childNodes[1].childNodes[colIndex].firstChild;
        return dataCell == null ? '' : dataCell.innerText.toLowerCase();
    };
}

function keyResult(colIndex) {
    return function(elem) {
        const strings = ['Error', 'Failed', 'Rerun', 'XFailed', 'XPassed',
            'Skipped', 'Passed'];
        return strings.indexOf(elem.childNodes[1].childNodes[colIndex].firstChild.data);
    };
}

function resetSortHeaders() {
    findAll('.sort-icon').forEach(function(elem) {
        elem.parentNode.removeChild(elem);
    });
    findAll('.sortable').forEach(function(elem) {
        const icon = document.createElement('div');
        icon.className = 'sort-icon';
        icon.textContent = 'vvv';
        elem.insertBefore(icon, elem.firstChild);
        elem.classList.remove('desc', 'active');
        elem.classList.add('asc', 'inactive');
    });
}

function toggleSortStates(elem) {
    //if active, toggle between asc and desc
    if (elem.classList.contains('active')) {
        elem.classList.toggle('asc');
        elem.classList.toggle('desc');
    }

    //if inactive, reset all other functions and add ascending active
    if (elem.classList.contains('inactive')) {
        resetSortHeaders();
        elem.classList.remove('inactive');
        elem.classList.add('active');
    }
}

function isAllRowsHidden(value) {
    return value.hidden == false;
}

function filterTable(elem) { // eslint-disable-line no-unused-vars
    const outcomeAtt = 'data-test-result';
    const outcome = elem.getAttribute(outcomeAtt);
    const classOutcome = outcome + ' results-table-row';
    const outcomeRows = document.getElementsByClassName(classOutcome);

    for(let i = 0; i < outcomeRows.length; i++){
        outcomeRows[i].hidden = !elem.checked;
    }

    const rows = findAll('.results-table-row').filter(isAllRowsHidden);
    const allRowsHidden = rows.length == 0 ? true : false;
    const notFoundMessage = document.getElementById('not-found-message');
    notFoundMessage.hidden = !allRowsHidden;
}
</script>
    <h1>report.html</h1>
    <p>Report generated on 15-Dec-2021 at 13:02:01 by <a href="https://pypi.python.org/pypi/pytest-html">pytest-html</a> v3.1.1</p>
    <h2>Environment</h2>
    <table id="environment">
      <tr>
        <td>Packages</td>
        <td>{"pluggy": "0.13.1", "py": "1.10.0", "pytest": "6.2.2"}</td></tr>
      <tr>
        <td>Platform</td>
        <td>Linux-4.15.0-159-generic-x86_64-with-Ubuntu-18.04-bionic</td></tr>
      <tr>
        <td>Plugins</td>
        <td>{"excel": "1.4.2", "html": "3.1.1", "metadata": "1.11.0"}</td></tr>
      <tr>
        <td>Python</td>
        <td>3.6.9</td></tr></table>
    <h2>Summary</h2>
    <p>0 tests ran in 1.25 seconds. </p>
    <p class="filter" hidden="true">(Un)check the boxes to filter the results.</p><input checked="true" class="filter" data-test-result="passed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="passed">0 passed</span>, <input checked="true" class="filter" data-test-result="skipped" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="skipped">0 skipped</span>, <input checked="true" class="filter" data-test-result="failed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="failed">0 failed</span>, <input checked="true" class="filter" data-test-result="error" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="error">6 errors</span>, <input checked="true" class="filter" data-test-result="xfailed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="xfailed">0 expected failures</span>, <input checked="true" class="filter" data-test-result="xpassed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="xpassed">0 unexpected passes</span>
    <h2>Results</h2>
    <table id="results-table">
      <thead id="results-table-head">
        <tr>
          <th class="sortable result initial-sort" col="result">Result</th>
          <th class="sortable" col="name">Test</th>
          <th class="sortable" col="duration">Duration</th>
          <th class="sortable links" col="links">Links</th></tr>
        <tr hidden="true" id="not-found-message">
          <th colspan="4">No results found. Try to check the filters</th></tr></thead>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">testcase/array/test_array.py::test_create_check::setup</td>
          <td class="col-duration">1.13</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>&gt;           assert pos.cli.start_system()[0] == True<br/><span class="error">E           AssertionError: assert False == True</span><br/><span class="error">E             +False</span><br/><span class="error">E             -True</span><br/><br/>testcase/conftest.py:163: AssertionError<br/><br/>During handling of the above exception, another exception occurred:<br/><br/>    @pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>            assert pos.cli.start_system()[0] == True<br/>        except Exception as e:<br/>            logger.error(e)<br/>&gt;           assert 0<br/><span class="error">E           assert 0</span><br/><br/>testcase/conftest.py:166: AssertionError<br/> -----------------------------Captured stdout setup------------------------------ <br/>2021-12-15 13:02:00,314 conftest.py pytest_runtest_protocol 63 INFO  :: ======================== START OF test_create_check ========================
2021-12-15 13:02:00,314 conftest.py pytest_runtest_protocol 66 INFO  :: Start Time : 12/15/2021, 13:02:00
2021-12-15 13:02:00,315 node.py __init__ 197 DEBUG  :: SSHclient constructor
2021-12-15 13:02:00,316 node.py connect 98 DEBUG  :: attempting to connect to: 127.0.0.1
2021-12-15 13:02:00,439 node.py connect 100 DEBUG  :: successfully connected to: 127.0.0.1
2021-12-15 13:02:00,440 node.py execute 236 DEBUG  :: Executing Command &quot;/root/pos-0.10.5/ibofos/bin/poseidonos-cli system start --json-res&quot; on host 127.0.0.1
2021-12-15 13:02:01,256 node.py execute 258 DEBUG  :: expected_exit_code is 0 and  current exit code is 127
2021-12-15 13:02:01,256 cli.py run_cli_command 136 INFO  :: ([], &#x27;exit code: 127&#x27;)
2021-12-15 13:02:01,256 cli.py run_cli_command 140 INFO  :: command execution completed in  : 0:00:00.816166 secs 
2021-12-15 13:02:01,257 cli.py run_cli_command 174 ERROR  :: Command Execution failed because of sequence item 0: expected str instance, list found
2021-12-15 13:02:01,257 cli.py core_dump 1300 INFO  :: ------------------------------------------ CLI HISTORY ------------------------------------------
2021-12-15 13:02:01,257 cli.py core_dump 1302 INFO  :: []
2021-12-15 13:02:01,257 cli.py core_dump 1306 INFO  :: -------------------------------------------------------------------------------------------------------
2021-12-15 13:02:01,257 node.py execute 236 DEBUG  :: Executing Command &quot;ps -aef | grep -i poseidonos&quot; on host 127.0.0.1
2021-12-15 13:02:01,381 node.py execute 258 DEBUG  :: expected_exit_code is 0 and  current exit code is 0
2021-12-15 13:02:01,382 node.py execute 236 DEBUG  :: Executing Command &quot;pkill -11 poseidonos&quot; on host 127.0.0.1
2021-12-15 13:02:01,443 node.py execute 258 DEBUG  :: expected_exit_code is 0 and  current exit code is 1
2021-12-15 13:02:01,443 cli.py core_dump 1312 ERROR  :: pkill -11 poseidonos for createing core dump file
2021-12-15 13:02:01,443 cli.py start_system 191 ERROR  :: failed due to CLI Error
2021-12-15 13:02:01,445 conftest.py start_pos 165 ERROR  :: assert False == True
  +False
  -True
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>2021-12-15 13:02:01,493 conftest.py pytest_runtest_logreport 88 INFO  :: ======================== Test Status : FAIL ========================
<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">testcase/array/test_array.py::test_unmount_pos_array::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>&gt;           assert pos.cli.start_system()[0] == True<br/><span class="error">E           AssertionError: assert False == True</span><br/><span class="error">E             +False</span><br/><span class="error">E             -True</span><br/><br/>testcase/conftest.py:163: AssertionError<br/><br/>During handling of the above exception, another exception occurred:<br/><br/>    @pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>            assert pos.cli.start_system()[0] == True<br/>        except Exception as e:<br/>            logger.error(e)<br/>&gt;           assert 0<br/><span class="error">E           assert 0</span><br/><br/>testcase/conftest.py:166: AssertionError<br/> -----------------------------Captured stdout setup------------------------------ <br/>2021-12-15 13:02:01,493 conftest.py pytest_runtest_protocol 69 INFO  :: End Time : 12/15/2021, 13:02:01
2021-12-15 13:02:01,494 conftest.py pytest_runtest_protocol 74 INFO  :: Execution Time : 0 minutes 1 seconds
2021-12-15 13:02:01,494 conftest.py pytest_runtest_protocol 78 INFO  :: ======================== END OF test_create_check ========================
2021-12-15 13:02:01,494 conftest.py pytest_runtest_protocol 63 INFO  :: ======================== START OF test_unmount_pos_array ========================
2021-12-15 13:02:01,494 conftest.py pytest_runtest_protocol 66 INFO  :: Start Time : 12/15/2021, 13:02:01
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>2021-12-15 13:02:01,503 conftest.py pytest_runtest_logreport 88 INFO  :: ======================== Test Status : FAIL ========================
<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">testcase/array/test_array.py::test_delete_array::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>&gt;           assert pos.cli.start_system()[0] == True<br/><span class="error">E           AssertionError: assert False == True</span><br/><span class="error">E             +False</span><br/><span class="error">E             -True</span><br/><br/>testcase/conftest.py:163: AssertionError<br/><br/>During handling of the above exception, another exception occurred:<br/><br/>    @pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>            assert pos.cli.start_system()[0] == True<br/>        except Exception as e:<br/>            logger.error(e)<br/>&gt;           assert 0<br/><span class="error">E           assert 0</span><br/><br/>testcase/conftest.py:166: AssertionError<br/> -----------------------------Captured stdout setup------------------------------ <br/>2021-12-15 13:02:01,503 conftest.py pytest_runtest_protocol 69 INFO  :: End Time : 12/15/2021, 13:02:01
2021-12-15 13:02:01,503 conftest.py pytest_runtest_protocol 74 INFO  :: Execution Time : 0 minutes 0 seconds
2021-12-15 13:02:01,503 conftest.py pytest_runtest_protocol 78 INFO  :: ======================== END OF test_unmount_pos_array ========================
2021-12-15 13:02:01,503 conftest.py pytest_runtest_protocol 63 INFO  :: ======================== START OF test_delete_array ========================
2021-12-15 13:02:01,504 conftest.py pytest_runtest_protocol 66 INFO  :: Start Time : 12/15/2021, 13:02:01
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>2021-12-15 13:02:01,512 conftest.py pytest_runtest_logreport 88 INFO  :: ======================== Test Status : FAIL ========================
<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">testcase/array/test_array.py::test_create_array_with_out_spare::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>&gt;           assert pos.cli.start_system()[0] == True<br/><span class="error">E           AssertionError: assert False == True</span><br/><span class="error">E             +False</span><br/><span class="error">E             -True</span><br/><br/>testcase/conftest.py:163: AssertionError<br/><br/>During handling of the above exception, another exception occurred:<br/><br/>    @pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>            assert pos.cli.start_system()[0] == True<br/>        except Exception as e:<br/>            logger.error(e)<br/>&gt;           assert 0<br/><span class="error">E           assert 0</span><br/><br/>testcase/conftest.py:166: AssertionError<br/> -----------------------------Captured stdout setup------------------------------ <br/>2021-12-15 13:02:01,512 conftest.py pytest_runtest_protocol 69 INFO  :: End Time : 12/15/2021, 13:02:01
2021-12-15 13:02:01,513 conftest.py pytest_runtest_protocol 74 INFO  :: Execution Time : 0 minutes 0 seconds
2021-12-15 13:02:01,513 conftest.py pytest_runtest_protocol 78 INFO  :: ======================== END OF test_delete_array ========================
2021-12-15 13:02:01,513 conftest.py pytest_runtest_protocol 63 INFO  :: ======================== START OF test_create_array_with_out_spare ========================
2021-12-15 13:02:01,513 conftest.py pytest_runtest_protocol 66 INFO  :: Start Time : 12/15/2021, 13:02:01
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>2021-12-15 13:02:01,521 conftest.py pytest_runtest_logreport 88 INFO  :: ======================== Test Status : FAIL ========================
<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">testcase/array/test_array.py::test_array_rebuild::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>&gt;           assert pos.cli.start_system()[0] == True<br/><span class="error">E           AssertionError: assert False == True</span><br/><span class="error">E             +False</span><br/><span class="error">E             -True</span><br/><br/>testcase/conftest.py:163: AssertionError<br/><br/>During handling of the above exception, another exception occurred:<br/><br/>    @pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>            assert pos.cli.start_system()[0] == True<br/>        except Exception as e:<br/>            logger.error(e)<br/>&gt;           assert 0<br/><span class="error">E           assert 0</span><br/><br/>testcase/conftest.py:166: AssertionError<br/> -----------------------------Captured stdout setup------------------------------ <br/>2021-12-15 13:02:01,521 conftest.py pytest_runtest_protocol 69 INFO  :: End Time : 12/15/2021, 13:02:01
2021-12-15 13:02:01,521 conftest.py pytest_runtest_protocol 74 INFO  :: Execution Time : 0 minutes 0 seconds
2021-12-15 13:02:01,522 conftest.py pytest_runtest_protocol 78 INFO  :: ======================== END OF test_create_array_with_out_spare ========================
2021-12-15 13:02:01,522 conftest.py pytest_runtest_protocol 63 INFO  :: ======================== START OF test_array_rebuild ========================
2021-12-15 13:02:01,522 conftest.py pytest_runtest_protocol 66 INFO  :: Start Time : 12/15/2021, 13:02:01
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>2021-12-15 13:02:01,531 conftest.py pytest_runtest_logreport 88 INFO  :: ======================== Test Status : FAIL ========================
<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">testcase/array/test_array.py::test_create_array_2_data_disks::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>&gt;           assert pos.cli.start_system()[0] == True<br/><span class="error">E           AssertionError: assert False == True</span><br/><span class="error">E             +False</span><br/><span class="error">E             -True</span><br/><br/>testcase/conftest.py:163: AssertionError<br/><br/>During handling of the above exception, another exception occurred:<br/><br/>    @pytest.fixture(scope=&quot;module&quot;)<br/>    def start_pos():<br/>        try:<br/>            global pos<br/>            pos = POS(<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;ip&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;username&quot;],<br/>                config_dict[&quot;login&quot;][&quot;target&quot;][&quot;password&quot;],<br/>                config_dict[&quot;login&quot;][&quot;paths&quot;][&quot;pos_path&quot;],<br/>            )<br/>            assert pos.cli.start_system()[0] == True<br/>        except Exception as e:<br/>            logger.error(e)<br/>&gt;           assert 0<br/><span class="error">E           assert 0</span><br/><br/>testcase/conftest.py:166: AssertionError<br/> -----------------------------Captured stdout setup------------------------------ <br/>2021-12-15 13:02:01,531 conftest.py pytest_runtest_protocol 69 INFO  :: End Time : 12/15/2021, 13:02:01
2021-12-15 13:02:01,531 conftest.py pytest_runtest_protocol 74 INFO  :: Execution Time : 0 minutes 0 seconds
2021-12-15 13:02:01,531 conftest.py pytest_runtest_protocol 78 INFO  :: ======================== END OF test_array_rebuild ========================
2021-12-15 13:02:01,531 conftest.py pytest_runtest_protocol 63 INFO  :: ======================== START OF test_create_array_2_data_disks ========================
2021-12-15 13:02:01,532 conftest.py pytest_runtest_protocol 66 INFO  :: Start Time : 12/15/2021, 13:02:01
<br/> ----------------------------Captured stdout teardown---------------------------- <br/>2021-12-15 13:02:01,540 conftest.py pytest_runtest_logreport 88 INFO  :: ======================== Test Status : FAIL ========================
<br/></div></td></tr></tbody></table></body></html>